src/pages/home/report/ReportAttachments.tsx

```diff
@@ -36,7 +36,7 @@ function ReportAttachments({route}: ReportAttachmentsProps) {
     });
     const [isLoadingApp] = useOnyx(ONYXKEYS.IS_LOADING_APP, {canBeMissing: false});
     const {isOffline} = useNetwork();
-    const fileName = route.params?.fileName;
+    const fileName = route.params?.originalFileName;
```

Comment:
This may be a dumb question but why there is the `orginalFileName` param when both route and param list suggest just `fileName`?

Reply:
Good question, I wasn't sure either. I think this is mostly a relic of some bug fix/feature for `AttachmentModal`.

We should definitely make this consistent and remove `originalFileName` in favor of `fileName`, but i would suggest to do that in a follow-up PR, because it would affect around ~60 files and we just reduced the scope of this initial PR